holoviews/plotting/bokeh/raster.py

```diff
@@ -156,10 +167,404 @@ def get_data(self, element, ranges, style):
                 sanitized = dimension_sanitizer(vdim.name)
                 data[sanitized] = ['-' if is_nan(v) else vdim.pprint_value(v)
                                    for v in aggregate.dimension_values(vdim)]
+
+        # Filter radial heatmap options
+        style = {k: v for k, v in style.items() if not
+                 any(g in k for g in RadialHeatMapPlot._style_groups.values())}
         return (data, {'x': x, 'y': y, 'fill_color': {'field': 'zvalues', 'transform': cmapper},
                        'height': 1, 'width': 1}, style)
 
 
+class RadialHeatMapPlot(CompositeElementPlot, ColorbarPlot):
```

Comment:
Between ``HeatMapPlot`` and ``RadialHeatMapPlot`` I wonder if it is worth introducing ``plotting/bokeh/heatmap.py`` (and the same for matplotlib).

Reply:
No objection, they aren't really rasters anyway.

Reply:
Agreed. I think it would be less confusing.